lib/pages/inbox.dart

```diff
@@ -379,7 +380,9 @@ class PrivateMessageTile extends HookWidget {
               TileAction(
                 delayedLoading: deleteDelayed,
                 icon: deleted.value ? Icons.restore : Icons.delete,
-                tooltip: deleted.value ? 'restore' : 'delete',
+                tooltip: deleted.value
+                    ? L10n.of(context).restore
+                    : L10n.of(context).delete,
```

Comment:
restore-delete not updated in more menus in comments

Reply:
Because these are capitalized, I am not sure what to do about those

Reply:
Isn't there a method on String `.toCapitalized()`?